src/Models/Workflow.php

```diff
@@ -245,8 +245,8 @@ public function serialize()
             'user' => $user,
             'status' => $this->getStatus(),
             'time' => sprintf('%ds', $elapsed_time),
-            'finished_at' => $this->get('finished_at'),
-            'started_at' => $this->get('started_at'),
```

Comment:
Now, these dates, at least, are currently in 'epoch' form. It would be a b/c break to switch them to a human-readable format here.

We could use a cell formatter to convert these to human-readable dates only for table formats, ignoring potential b/c breaks on the notion that anyone parsing output values should be using json/yml format instead. 

The other alternative would be to make twice as many date fields (one for machine-readable, one for human-readable). It would be difficult to do that without breaking b/c, though, as you'd probably want to switch the default display to human-readable, but the current default is machine-readable. I think that a cell formatter would be safer. I could push a commit here to show how that would be done if you were interested in pursuing that option.

Reply:
This change is being released at 2.0, so breaking changes are permissible.

Reply:
I'm open to improvement breaking changes right now because this is all being released as 2.0.

I switched a lot of things around and the `serialize()` functions now emit only Unix datetimes but I added renderer functions onto the StructuredLists to handle all the formatting and also the conversion of booleans to `'true'` or `'false'`. Just updating the tests.